Sources/Private/CoreAnimation/Layers/TextLayer.swift

```diff
@@ -23,6 +23,22 @@ final class TextLayer: BaseCompositionLayer {
     fatalError("init(coder:) has not been implemented")
   }
 
+  override func setupAnimations(context: LayerAnimationContext) throws {
+    try super.setupAnimations(context: context)
+
+    // Question 1: Which keypath to pass?
+    _ = try context.currentKeypath.keys.map { key in
+      // Question 2: Where do we get the text layer text?
```

Comment:
`try textLayerModel.text.exactlyOneKeyframe(context: context, description: "text layer text").text` is text value displayed in the text layer.

Due to limitations of Core Animation text rendering, this text layer only supports displaying a static text value over the entire animation. That's why we extract a single string from the keyframes using `exaxtlyOneKeyframe()`.

Reply:
Updated the method in [5055f10](https://github.com/airbnb/lottie-ios/pull/1723/commits/5055f10ae3a5f9a66bda958d9b2bb3e4768d3129) 🟡 

> Due to limitations of Core Animation text rendering, this text layer only supports displaying a static text value over the entire animation. That's why we extract a single string from the keyframes using exaxtlyOneKeyframe().

Do you mean that text won't be able to be animated in the end anyway, or that just the text can't be changed during the animation? Like we can't start animation with text "Hell.." and finish it with "Hello!" 🤔